index.html

```diff
@@ -0,0 +1,145 @@
+										<header class="major">
+											<h2>Dani the Dev</h2>
+										</header>
+										<p>I am a recent computer science graduate with my degree in Web Development and a self taught graphc designer.
+											I am looking to advance my knowledge with Lambda in hopes to become a full stack developer
+											or software engineer.
+										</p>
+										
+									</div>
+									<span class="image"><img src="images/pic01.jpg" alt="" /></span>
+								</div>
+							</section>
+
+						<!-- First Section -->
+							<section id="first" class="main special">
+								<header class="major">
+									<h2>Web Portfolio</h2>
+								</header>
+								<ul class="features">
+									<li>
+										<span class="icon major style1 fa-code"></span>
+										<h3>Responsive Web Design</h3>
+									</li>
+									<li>
+										<span class="icon major style3 fa-copy"></span>
+										<h3>Javascript</h3>
+									</li>
+									<li>
+										<span class="icon major style5 fa-diamond"></span>
+										<h3>HTML5/CSS</h3>
+									</li>
+								</ul>
+								<footer class="major">
+									<ul class="actions special">
+										<li><a href="https://github.com/droneal11015" class="button">See My Work</a></li>
+									</ul>
+								</footer>
+							</section>
+
+						<!-- Second Section -->
+							<section id="second" class="main special">
+								<header class="major">
+									<h2>Graphic Design</h2>
+									<p> From custom logos to ellaborate illustrations, I can design something to fit all your needs. </p>
+								</header>
+								
+								<footer class="major">
+									<ul class="actions special">
+										<li><a href="https://www.facebook.com/digitalyelldesigns/" class="button">My Art</a></li>
+									</ul>
+								</footer>
+							</section>
+
+						<!-- Get Started -->
+							<section id="cta" class="main special">
+								<header class="major">
+									<h2>Let's make something beautiful!</h2>
+									<p>Contact me today to get a free quote for your business needs! </p>
+								</header>
+								
+							</section>
+
+					</div>
+
+				<!-- Footer -->
+					<footer id="footer">
+						<section>
+							<h2></h2>
+							
+						</section>
+						<section>
+							<h2>Contact Info</h2>
+							<dl class="alt">
+								<dt>Address</dt>
+								<dd>273 Overland Way &bull; Gray, GA 31032  &bull; USA</dd>
+								<dt>Phone</dt>
+								<dd>(478) 607-9358</dd>
+								<dt>Email</dt>
+								<dd><a href="#">contact@danionealdev.com</a></dd>
+							</dl>
+							<ul class="icons">
+								<li><a href="facebook.com/digitalyelldesigns" class="icon fa-facebook alt"><span class="label">Facebook</span></a></li>
+								<li><a href="https://www.instagram.com/digitalyelldesigns/" class="icon fa-instagram alt"><span class="label">Instagram</span></a></li>
+								<li><a href="https://github.com/droneal11015" class="icon fa-github alt"><span class="label">GitHub</span></a></li>
```

Comment:
Heads-up for your Facebook link: you'll also want to add "https://www." at the start of the `href` attribute value to make sure that the link will always redirect to the actual Facebook page that you want.